functions/src/index.js

```diff
@@ -12,7 +12,7 @@ const config = require("./config/config").getConfig(env);
 
 admin.initializeApp({ 
     credential: admin.credential.cert(config.serviceAccount),
-    storageBucket: "wpa-be-app-dev.appspot.com" 
+    storageBucket: "wpa-be-app.appspot.com" 
```

Comment:
Why did we change this?

Reply:
nevermind, i thought this was the config file...I guess we should make this dependant on the env so we don't have to manually change it for dev/prod